### PR TITLE
add M.BIO to IPCC2006_PRIMAP

### PR DESCRIPTION
**Pull request by crdanielbusch**
# Pull request

Please confirm that this pull request has done the following:

- [x] Tests added
- [x] Documentation added (where applicable)
- [x] Changelog snippet in a `.rst` file in the directory `changelog_unreleased` added – remember to start with a `*` to make it a bullet point

## Description

Add the M.BIO category to IPCC2006_PRIMAP

### CODE REVIEW

File: climate_categories/data/IPCC2006_PRIMAP.yaml [hunk 1]
```diff
@@ -3204,6 +3204,7 @@ categories:
     - 3A
     info:
       gases:
+      - N2O
```
Comment (crdanielbusch) [inline climate_categories/data/IPCC2006_PRIMAP.yaml:3207]: This appeared up when I ran `data_generation/IPCC2006_PRIMAP.py`.

Comment (mikapfl) [reply climate_categories/data/IPCC2006_PRIMAP.yaml:3207]: Yeah, I just missed re-generating after fixing this.

**Review by mikapfl — APPROVED** [review]

**Review by JGuetschow — APPROVED** [review]
Looks good

**Comment by crdanielbusch** [comment]
Thanks for the review. Merging thid

**Comment by crdanielbusch** [comment]
Forgot to add the changelog. Can you have a look at the latest commit?

**Review by mikapfl — APPROVED** [review]

**Pull request merged**